test-zeekoe.py

```diff
@@ -215,7 +257,7 @@ def main():
     parser.add_argument("command", help="", nargs="?", default="list")
     parser.add_argument("--path", help="path to create configs", default="./dev")
     parser.add_argument("--network", help="select the type of network", default=SANDBOX)
-    parser.add_argument("--self-delay", "-t", type=int, help="self-delay for closing transactions", default="1")
+    parser.add_argument("--self-delay", "-t", type=int, help="self-delay for closing transactions", default="120")
```

Comment:
Nonblocking: Could the helptext include a list of valid commands along with a brief description of what they do? This would let people use this tool without needing to read the source code.